### PR TITLE
Add mpi and mpiCredentials

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: spec/definitions/MpiCredentials/CardinalCommerce.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+description: CardinalCommerce Mpi Config
```
Comment (adamaltman) [inline spec/definitions/MpiCredentials/CardinalCommerce.yaml]: Below you use the word `Credentials` -- adjust `Config` here to be `Credentials` to be consistent.

File: spec/definitions/MpiCredentials/Other.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+description: Other
+allOf:
+  -
+    $ref: "#/definitions/Mpi"
+  -
```
Comment (slavcodev) [inline spec/definitions/MpiCredentials/Other.yaml]: I think you it is redundant merging.

File: spec/definitions/MpiCredentials/PayvisionMpi.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+description: Payvision Integrated
+allOf:
+  -
+    $ref: "#/definitions/Mpi"
+  -
```
Comment (slavcodev) [inline spec/definitions/MpiCredentials/PayvisionMpi.yaml]: Extra item

**Comment by Zaphell** [comment]
Looks good to me :+1:

**Pull request merged**